libc/src/__support/time/windows/qpc.h

```diff
@@ -0,0 +1,35 @@
+//===--- Cached QPC Frequency  ----------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/__support/CPP/atomic.h"
+#include "src/__support/common.h"
+
+#define WIN32_LEAN_AND_MEAN
+#define NOMINMAX
+#include <Windows.h>
+
+namespace LIBC_NAMESPACE_DECL {
+namespace qpc {
```

Comment:
`qpc` isn't an acronym I'm familiar with in this context. I'd recommend expanding it for readability.

Reply:
fixed